packages/blueprints-integration/src/rundown.ts

```diff
@@ -343,20 +361,24 @@ export interface IBlueprintPieceGeneric<TMetadata = unknown> {
 		outTransition?: PieceTransition
 	}
 
-	/** Duration to preroll/overlap when running this adlib */
-	adlibPreroll?: number
+	/**
+	 * How long this piece needs to prepare its content before it will have an effect on the output.
+	 * This allows for flows such as starting a clip playing, then cutting to it after some ms once the player is outputting frames.
+	 */
+	prerollDuration?: number
+
 	/** Whether the adlib should always be inserted queued */
 	toBeQueued?: boolean
 	/** Array of items expected to be played out. This is used by playout-devices to preload stuff.
 	 * @deprecated replaced by .expectedPackages
 	 */
 	expectedPlayoutItems?: ExpectedPlayoutItemGeneric[]
-	/** When queued, should the adlib autonext */
-	adlibAutoNext?: boolean
-	/** When queued, how much overlap with the next part */
-	adlibAutoNextOverlap?: number
-	/** When queued, block transition at the end of the part */
-	adlibDisableOutTransition?: boolean
+	// /** When queued, should the adlib autonext */
+	// adlibAutoNext?: boolean
```

Comment:
Remove these?

Reply:
theyve already been removed in r39, will be tidied as part of merging r39 back into this branch